packages/app/src/components/DimensionsPanel/Dialogs/DialogManager.js

```diff
@@ -130,6 +130,8 @@ export class DialogManager extends Component {
         }
     }
 
+    onClose = () => this.props.closeDialog(null);
```

Comment:
rename to "closeDialog" or just "close"?

Reply:
Renamed to `onClose` for maintaining the naming conventions.